### PR TITLE
Rename mapbox to maplibre in typings file

### PR DESCRIPTION
**Pull request by HarelM**
## Launch Checklist

This PR is to update the typings definition to reflect the changes in the maplibre rename of mapbox elements. See #27

 - [x] confirm your changes do not include backports from Mapbox projects (unless with compliant license)
 - [x] briefly describe the changes in this PR
 - [ ] ~~include before/after visuals or gifs if this PR includes visual changes~~
 - [ ] ~~write tests for all new functionality~~
 - [ ] ~~document any changes to public APIs~~
 - [ ] ~~post benchmark scores~~
 - [ ] ~~manually test the debug page~~
 - [x] apply changelog label ('bug', 'feature', 'docs', etc) or use the label 'skip changelog' - already in changelog
 - [ ] ~~add an entry inside this element for inclusion in the `maplibre-gl-js` changelog: `<changelog></changelog>`~~

### CODE REVIEW

**Comment by HarelM** [comment]
Related to a code review comment as part of #30. If #30 is merged there's no real reason for this PR.

File: src/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for MapLibre GL JS 1.14.0-rc.1
+// Type definitions for MapLibre GL JS 1.14.0
```
Comment (wipfli) [inline src/index.d.ts:1]: Is it necessary to have the version in the comment? This is something we might often step over and forget to update.

File: src/index.d.ts [hunk 2]
```diff
@@ -760,6 +760,10 @@ declare namespace mapboxgl {
 
     export type TransformRequestFunction = (url: string, resourceType: ResourceType) => RequestParameters;
 
+    export type ResponseCallback<T> = (error?: Error, data?: T, cacheControl?: string, expires?: string) => void;
```
Comment (wipfli) [inline src/index.d.ts:763]: What is the context of these two?

File: src/index.d.ts [hunk 3]
```diff
@@ -760,6 +760,10 @@ declare namespace mapboxgl {
...
+
+    export type Cancelable = { cancel: () => void };
+
```
Comment (wipfli) [inline src/index.d.ts:766]: What is the context of these two?

**Comment by wipfli** [comment]
Thanks @HarelM for the clean-up. Looks good to me ✅

**Review by wipfli — APPROVED** [review]

File: src/index.d.ts (hunk 1 above)
Comment (HarelM) [reply src/index.d.ts:1]: Maybe not, not sure, it might be redundant now that's it's inside this repo...

File: src/index.d.ts (hunk 3 above)
Comment (HarelM) [reply src/index.d.ts:766]: Missing types in general. I took the full file from #30 and forgot to remove these two... I can remove them if needed...

**Comment by HarelM** [comment]
So, what's the next step in order to merge this in? (and hopefully release a version with this to npm... :-))

**Comment by wipfli** [comment]
I guess just click "squash and merge". You have write access to this repo, right?

**Comment by HarelM** [comment]
Is there a way for me to publish an npm pakcage as well? Some sort of github action? Other CI I can trigger?

**Comment by wipfli** [comment]
Please read https://github.com/maplibre/maplibre-gl-js/blob/main/RELEASE-PROCESS.md

**Comment by wipfli** [comment]
You can go ahead and create a release candidate on NPM by pushing a tag of the form `v1.14.1-rc.1`. Please try to push such a tag including the `-rc.1`. Then we will see if you have the rights to do so.

**Comment by wipfli** [comment]
And of course let's first publish release candidates and let's wait with actual releases.

**Comment by HarelM** [comment]
Sure thing. I'll do it later tonight. I need to commit the pakckage.json file with the version change for rc as well, right?

**Comment by wipfli** [comment]
Yes please. NPM will only see the version in the tag, but the docs website will use the version in `package.json`. In the last release we forgot to change this an this is why we got:

image

**Comment by wipfli** [comment]
https://maplibre.org/maplibre-gl-js-docs/api/

**Pull request merged**